.github/workflows/handler-test.yml

```diff
@@ -60,6 +78,36 @@ jobs:
           echo "Curling \`health_check_url\` for a return status of 200..."
           while ! curl -sfS --max-time 5 $( terraform output health_check_url ); do sleep 5; done
 
+      - name: Retrieve TFE URL
+        id: retrieve-tfe-url
+        run: |
+          terraform output -no-color -raw tfe_url
+
+      - name: Retrieve IACT Token
+        id: retrieve-iact-token
+        run: |
+          curl $( terraform output -no-color -raw iact_url )
+
+      - name: Create Admin in TFE
+        id: create-admin
+        run: |
+          curl \
+            --header "Content-Type: application/json" \
+            --data '{ "username": "tester", "email": "tf-onprem-team@hashicorp", "password": "${{ secrets.TFE_PASSWORD }}" }' \
+            $( terraform output -no-color -raw initial_admin_user_url )?token=${{ steps.retrieve-iact-token.stdout }} \
+            | jq --raw-output '.token'
+
+      - name: Run k6 Smoke Tests
+        id: run-smoke-tests
+        working-directory: *K6_WORK_DIR
+        env:
+          K6_PATHNAME: "./k6"
```

Comment:
We will need to update the load test repository Makefile to use this variable; alternatively, looks like we may be able to edit the PATH: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path.

Reply:
I think it'd be better to add it to the `hashicorp/tfe-load-test` repository. The docs you pointed to seem to affect subsequent paths and would mean we'd have to go into the test path and then also go out of the test path.